pkg/controller-gen/generators/client_generator.go

```diff
@@ -95,6 +95,7 @@ func (cg *ClientGenerator) typesGroupVersionDocPackage(name *types.Name, gv sche
 	p.HeaderComment = []byte(fmt.Sprintf(`
 %s
 
+// +k8s:openapi-gen=true
```

Comment:
We should make this dependent on the GenerateOpenAPI flag.

 I thought we were okay with just doing the same as deepcopy but I misunderstood that deepcopy is generated when this generator runs..

It will also prevent having a huge amount of file changes in rancher/rancher when we update wrangler. (And also will prevent any change in behavior due to that new directive)

Could you make that change? Sorry about this.